src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,77 @@
+import { useSearchParams } from 'react-router-dom';
+import cn from 'classnames';
+import { SearchLink } from './SearchLink';
+
+const CENTURIES = ['16', '17', '18', '19', '20'];
+
+enum Sex {
+  m = 'm',
+  f = 'f',
+}
```

Comment:
Move it to separate file
```suggestion
enum Sex {
  Male = 'm',
  Female = 'f',
}
```